### PR TITLE
get_mask for NmtFields

### LINKED ISSUE #139: NmtField does not have method to return mask
NmtFields can return maps with `f.get_maps()`, but there is no method to get the mask. It'd be very handy to have such a method.

### PR DESCRIPTION
**Pull request by damonge**
Closes #139

### CODE REVIEW

File: pymaster/tests/test_field.py [hunk 1]
```diff
@@ -42,6 +42,16 @@ def __init__(self):
 FT = FieldTester()
 
 
+def test_field_get_mask():
+    nside = 32
+    npix = hp.nside2npix(nside)
+    mp = np.random.randn(1, npix)
+    msk = np.random.rand(npix)
+    f = nmt.NmtField(msk, mp, n_iter=0)
```
Comment (carlosggarcia) [inline pymaster/tests/test_field.py:50]: If the field is `lite`, does `get_mask` work, as well? (I guess yes, but just in case)

**Review by carlosggarcia — CHANGES_REQUESTED** [review]
Just a very minor question.

File: pymaster/tests/test_field.py (hunk 1 above)
Comment (damonge) [reply pymaster/tests/test_field.py:50]: Yes, fields always need a mask https://github.com/LSSTDESC/NaMaster/blob/8b95669a868ac75cd2a065677b6b4ab525781a37/src/nmt_field.c#L120

**Review by carlosggarcia — APPROVED** [review]
LGTM

File: pymaster/tests/test_field.py (hunk 1 above)
Comment (carlosggarcia) [reply pymaster/tests/test_field.py:50]: Good. Thanks!

**Review by carlosggarcia — APPROVED** [review]
LGTM 2

**Pull request merged**